.github/workflows/prettierrc.yml

```diff
@@ -0,0 +1,26 @@
+name: Format the code
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    name: Format Files
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "16"
+      - name: Prettier
+        run: npx prettier --write **/*.js
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - uses: stefanzweifel/git-auto-commit-action@v4
```

Comment:
I think that it will be better to add it as a `pre-commit` hook and verify the formatting when opening a pull request instead of formatting it in the CI.

Reply:
Ok ✌🏼